aclk/agent_cloud_link.c

```diff
@@ -1400,18 +868,25 @@ void *aclk_main(void *ptr)
         }
     }
 
+    query_threads.count = appconfig_get_number(&cloud_config, CONFIG_SECTION_GLOBAL, "query thread count", 2);
```

Comment:
I feel that choosing 2 as the default number is too little. A lot of people have systems that could take advantage of more threads. Even raspberry pi has 4-cores these days.

Reply:
Well currently even 1 thread is enough. This is the motive of default being 2. Until we see need for it (longer queries) I felt having 4 threads just sitting there is not necessary (even if they add little overhead)

Reply:
This should probably be in `netdata.conf` and not in the machine generated file in `/var/lib/cloud.d`.

Reply:
We should also have a line in a document somewhere that describes this so that users know how to configure this.

Reply:
thanks, done